src/pages/Flavors.js

```diff
@@ -66,6 +66,40 @@ export class Flavors extends Component {
     } else {
       this.setState({ [name]: checked });
     }
+  }*/
+
+  handleStashToggle() {
+    const { stash } = this.props;
+    const { holdings: stashMap } = this.state;
+    const holdings = {};
+
+    const { stashToggle } = this.state;
+
+    if (!stashMap) {
+      stash.map(flavor => {
+        holdings[flavor.flavorId] = true;
+      });
+
+      this.setState({ stashToggle: !stashToggle, holdings });
+    } else {
+      this.setState({ stashToggle: !stashToggle });
+    }
+  }
+
+  stashToggle() {
+    const { stashToggle } = this.state;
+
+    return (
+      <ToggleButton
+        value={stashToggle}
+        onClick={e => this.handleStashToggle(e)}
+        buttonProps={{
+          title: stashToggle ? 'Enable Flavor Stash' : 'Disable Flavor Stash'
```

Comment:
I think this might be backwards - if stashToggle is true then clicking the button would disable it, right?